pkg/tsdb/cloudmonitoring/utils.go

```diff
@@ -86,6 +87,11 @@ func doRequestPage(ctx context.Context, logger log.Logger, r *http.Request, dsIn
 		return cloudMonitoringResponse{}, err
 	}
 
+	defer func() {
+		err := res.Body.Close()
+		logger.Warn("failed to close response body", "error", err)
```

Comment:
just checking if it's it on purpose `Warn` instead of `Error`

Reply:
Yes, I don't think it's necessarily actionable that a response body wasn't properly closed. If it happens a lot it's a problem, if it happens once or twice, meh.

Reply:
👍 should we change it then for the rest as well?

Reply:
hm, yeah